ts/test/trycp/conductor.ts

```diff
@@ -306,28 +317,8 @@ test("TryCP Conductor - install app with roles settings", async (t) => {
 
   const app = await conductor.installApp(
     {
-      type: "bundle",
-      value: {
-        manifest: {
-          manifest_version: "1",
-          name: "app",
-          roles: [
-            {
-              name: ROLE_NAME,
-              provisioning: {
-                strategy: CellProvisioningStrategy.Create,
-                deferred: false,
-              },
-              dna: {
-                path: realpathSync(FIXTURE_DNA_URL),
-                modifiers: { network_seed: "some_seed" },
-              },
-            },
-          ],
-          membrane_proofs_deferred: true,
```

Comment:
Same here, no need for `membrane_proofs_deferred: true` so installing from path instead.